src/Utilities/cachingoptimizer.jl

```diff
@@ -393,9 +395,21 @@ end
 
 function MOI.add_constraint(
     m::CachingOptimizer,
-    func::MOI.AbstractFunction,
-    set::MOI.AbstractSet,
-)
+    func::F,
+    set::S,
+) where {F<:MOI.AbstractFunction,S<:MOI.AbstractSet}
+    if m.state != NO_OPTIMIZER && !MOI.supports_constraint(m.optimizer, F, S)
```

Comment:
Check that the optimizer is not already a `LazyBridgeOptimizer`

Reply:
Done